owtf/webapp/src/containers/OtpPage/index.js

```diff
@@ -68,62 +62,60 @@ export class OtpPage extends React.Component {
 
   render() {
     return (
-      <Pane marginY={100} data-test="otpPageComponent">
-        <Pane
-          justifyContent="center"
-          width="35%"
-          elevation={1}
-          margin="auto"
-          padding={5}
-        >
-          <Heading
-            size={700}
-            textAlign="center"
-            marginBottom={20}
-            paddingTop={20}
-          >
+      <div className="otpPageContainer" data-test="otpPageComponent">
+        <div className="otpPageContainer__otpComponentContainer">
+          <div className="otpPageContainer__otpComponentContainer__brandLogoContainer">
+            <img src={logo} alt="brand-logo" />
+          </div>
+
+          <h2 className="otpPageContainer__otpComponentContainer__heading">
             We sent a code
```

Comment:
Can you change the "we sent a code" to "A code was sent to your email" or something like that.